src/components/advanced-range-control/index.js

```diff
@@ -69,15 +70,26 @@ const AdvancedRangeControl = props => {
 		placeholderRender = null
 	}
 
+	const dynamicContentProps = useDynamicContentControlProps( {
+		value: typeof props.value === 'undefined' ? value : props.value,
+		onChange: typeof props.onChange === 'undefined' ? onChange : props.onChange,
+		isFormatType: propsToPass.isFormatType,
```

Comment:
We can omit the `isFormatType` to make it always false. From what I know this option makes it so that the dynamic content can be multiple instances, or allows part of the value to be dynamic content, both cases are usually for text content. But for the range control, the value is always a whole number so we can not use this.
```suggestion
		isFormatType: propsToPass.isFormatType,
```